stubservice/stubhandlers/stubhandler_test.go

```diff
@@ -63,10 +63,15 @@ func TestStoragePathEscape(t *testing.T) {
 	if res != "---Firefox58-4---" {
 		t.Errorf("String not properly escaped: %s", res)
 	}
+
+	res = storagePathEscape("")
+	if res != "-" {
+		t.Errorf("storagePathEscape should return '-' but returned '%s'", res)
+	}
 }
```

Comment:
I don't remember having since that before, why is this needed?

Reply:
I stumbled upon this error while debugging some of my changes. The added error log helped me find where the problem was. Definitely "out of context" for this patch, but since it improved overall coverage, I left it in. I could remove it if you want.